### DISCUSSION TITLE
HuggingFace Transformers

### DISCUSSION THREAD

**Discussion by chris-boson** [Lightning-AI/pytorch-lightning#6459] [code help: NLP / ASR / TTS]
Up until end of last year lightning trainer was officially supported by Huggingface transformers, but recently it was dropped in favor of their own trainer and datasets API. Now it only exists in some of the [research projects](https://github.com/huggingface/transformers/tree/master/examples/research_projects/seq2seq-distillation).

Do we know the reasoning for this? Are there any plans for an alignment in the future? I would prefer to continue using lightning for all my ML projects rather than having to modify my toolchain for individual projects.

**Comment by edenlightning** [comment]
Hey Chris thanks for reaching out! We are aware that the experience is less ideal now, and we will look into how to make it smoother.
